src/main/java/com/netflix/blitz4j/LoggingContext.java

```diff
@@ -172,6 +182,37 @@ public LocationInfo generateLocationInfo(LoggingEvent event) {
         return locationInfo;
     }
 
+    private boolean isUsingNFPatternLayout(LoggingEvent event) {
+        final Category logger = event.getLogger();
+        return logger != null && isUsingNFPatternLayout(logger.getAllAppenders());
+
+    }
+
+    private boolean isUsingNFPatternLayout(Enumeration enumeration) {
```

Comment:
Do you expect the `Category` under getLogger to be something I can do a == or equals on and that it won't change in a way that it would cause issues for me?